### ISSUE TITLE
refactor story features to use "story" instead of "status"

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2935]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.49.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.49.0...4.49.1) (2022-11-01)

- Release 4.49.1 [`fe23bd1`](https://github.com/open-wa/wa-automate-nodejs/commit/fe23bd15fdd00f061e43bc962f6b7689674bceba)
- 🔖 manual version bump 4.49.0 [`2f6162e`](https://github.com/open-wa/wa-automate-nodejs/commit/2f6162e54f031b4b07d71d9a6101fee3cda904dd)
- ✨ `onStory` now avail for restricted licenses #2935 [`cee1754`](https://github.com/open-wa/wa-automate-nodejs/commit/cee17548f83fb169ceb177ad51714cebf9d45025)
- 💚 fix ci [`21b6abc`](https://github.com/open-wa/wa-automate-nodejs/commit/21b6abc5628c09a08e657797b8b87acdf5814a9b)
